pandas/io/sql.py

```diff
@@ -608,6 +608,8 @@ def read_sql(
     Returns
     -------
     DataFrame or Iterator[DataFrame]
+        Returns a single DataFrame if applicable, or an Iterator
+        containing multiple DataFrames.
```

Comment:
The type is already clear from the previous line, can you explain in the description what is being returned, not expanding on the type.